Makefile

```diff
@@ -285,6 +285,16 @@ _push_litmus_git_app_checker:
 
 litmus-git-app-checker: deps _build_litmus_git_app_checker _push_litmus_git_app_checker
 
+_build_litmus_pg_jmeter:
```

Comment:
Should we also add it in `.github/workflows` ?

Reply:
Sure @uditgaurav